src/app/[locale]/(pages)/expiredMagicLink/page.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+import { Button, Grid, Typography } from '@mui/material';
+import { CieIcon } from '@pagopa/mui-italia/dist/icons/CieIcon';
+import { SpidIcon } from '@pagopa/mui-italia/dist/icons/SpidIcon';
+import { useTranslations } from 'next-intl';
+import { useState } from 'react';
+import { SpidLevels } from '../../_component/selectIdp/idpList';
+import { SelectIdp } from '../../_component/selectIdp/selectIdp';
+import Timeout from '../../_icons/timeout';
+import { COMMON_PADDING_HERO } from '../../_utils/styles';
+
+const ExpiredMagicLink = () => {
+  const t = useTranslations('ioesco');
+  const [openDialog, setOpenDialog] = useState<boolean>(false);
+
+  const spidLevel: SpidLevels = {
+    type: 'L2',
+  };
+
+  return (
+    <Grid sx={COMMON_PADDING_HERO} container bgcolor="background.default">
+      <Grid item xs={12} justifySelf={'center'}>
+        <Grid container justifyContent="center">
+          <Grid item xs={12} textAlign={'center'} pb={4}>
+            {<Timeout />}
+          </Grid>
+          <Grid item xs={12} pb={1}>
+            <Typography
+              variant="h4"
+              py={0}
+              px={0}
+              color="textPrimary"
+              sx={{
+                textAlign: 'center',
+              }}
+            >
+              Il link è scaduto
+            </Typography>
+          </Grid>
+          <Grid item xs={12} pb={4}>
+            <Typography
+              variant="body1"
+              py={0}
+              px={0}
+              color="textPrimary"
+              sx={{
+                textAlign: 'center',
+              }}
+            >
+              Per bloccare l’accesso a IO è necessario accedere al portale con le tue credenziali
```

Comment:
Why you don't use the translations? The key is "lockiolog" into "error"

Reply:
added missing translation